tests/test_unit/test_tile.py

```diff
@@ -1,4 +1,106 @@
-from brainglobe_stitch.tile import Tile
+from typing import List, Tuple
```

Comment:
Don't know if you only use `tox` for testing, but with `pytest` the tests do not run. I think the `pyproject.toml` file needs a change:
```
[tool.pytest.ini_options]
addopts = ". --cov=brainglobe_stitch"
```

Reply:
I can't seem to replicate this. What's the error?

Reply:
No idea, now it works without any issue 🤷🏻‍♀️